titus-server-master/src/main/java/com/netflix/titus/master/scheduler/ContainerFailureBasedAgentQualityTracker.java

```diff
@@ -32,6 +32,38 @@
 
     private static final Logger logger = LoggerFactory.getLogger(ContainerFailureBasedAgentQualityTracker.class);
 
+    private static final int UNDEFINED = -1;
+
+    /*
+     * Error weights. Each weight is a value between 0.1 and 0.5, or 0 if an error should be ignored.
+     * This constraint is due to the accumulative effect. We do not want to overreact on a single failure.
+     */
+
+    /**
+     * A container loss before it is started may indicate potential agent issue. We give it a middle score value.
+     */
+    private static final double LOST_BEFORE_STARTED_WEIGHT = 0.3;
+
+    /**
+     * A container loss after it was started is less likely to be an agent issue. We give it the lowest score.
+     */
+    private static final double LOST_AFTER_STARTED_WEIGHT = 0.1;
+
+    /**
+     * A container starting error due to an agent local problems. We give it a highest score.
+     */
+    private static final double LOCAL_STARTING_ERROR_WEIGHT = 0.5;
+
+    /**
+     * Not agent related issue us not counted as an error.
```

Comment:
Issue not related to agent; not counted as error

Reply:
Fixed